terraform/modules/data-share-service/rds.tf

```diff
@@ -34,6 +34,31 @@ resource "aws_rds_cluster" "rds_postgres_cluster" {
   }
 }
 
+data "aws_iam_policy_document" "rds_enhanced_monitoring_assume_role" {
```

Comment:
What does enhanced monitoring give us?

Reply:
Gives us access to better monitoring of the actual DB that's running
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.overview.html